contracts/BuyBackBurner.sol

```diff
@@ -55,12 +74,19 @@ contract BuyBackBurner {
     }
 
     /// @dev BuyBackBurner initializer.
-    function initialize() external{
+    /// @param payload Initializer payload.
+    function _initialize(bytes memory payload) internal virtual;
```

Comment:
This is needed to initialize the rest of required params for the `BuyBackBurner`.